packages/tasit-sdk/package.json

```diff
@@ -16,12 +17,21 @@
     "url": "git+https://github.com/tasitlabs/tasitsdk.git"
   },
   "scripts": {
-    "test": "echo \"Error: run tests from root\" && exit 1"
+    "prepublish": "npx babel src --out-dir lib --ignore **/*.test.js",
+    "test": "npx mocha src/index.test.js --require @babel/register"
   },
   "bugs": {
     "url": "https://github.com/tasitlabs/tasitsdk/issues"
   },
   "dependencies": {
-    "tasit-account": "^0.0.1"
```

Comment:
@marcelomorgado So you're thinking we'll reintroduce this dependency once we publish it, right?